app/api/business/supplier_business.py

```diff
@@ -33,14 +33,19 @@ def get_supplier_messages(code, skip_application_check):
             'id': 'SB001'
         })
 
+    if skip_application_check is False:
```

Comment:
Is this deliberately guarding against if `skip_application_check` is `None`? If not, just `if not skip_application_check:` might be more reliable.

Reply:
it is guarding against None

Reply:
updated